validator/client/validator.go

```diff
@@ -569,6 +577,36 @@ func (v *validator) isAggregator(ctx context.Context, committee []types.Validato
 	return binary.LittleEndian.Uint64(b[:8])%modulo == 0, nil
 }
 
+// isSyncCommitteeAggregator checks if a validator in an aggregator of a subcommittee for sync committee.
+func (v *validator) isSyncCommitteeAggregator(ctx context.Context, slot types.Slot, pubKey [48]byte) (bool, error) {
+	res, err := v.validatorClient.GetSyncSubcommitteeIndex(ctx, &ethpb.SyncSubcommitteeIndexRequest{
+		PublicKey: pubKey[:],
+		Slot:      slot,
+	})
+	if err != nil {
+		return false, err
+	}
+
+	modulo := uint64(1)
+	count := params.BeaconConfig().SyncCommitteeSize / params.BeaconConfig().SyncCommitteeSubnetCount / params.BeaconConfig().TargetAggregatorsPerSyncSubcommittee
+	if count > 1 {
+		modulo = count
+	}
+
+	for _, index := range res.Indices {
+		sig, err := v.signSyncSelectionData(ctx, pubKey, index, slot)
+		if err != nil {
+			return false, err
+		}
+		b := hashutil.Hash(sig)
+		if binary.LittleEndian.Uint64(b[:8])%modulo == 0 {
+			return true, nil
+		}
```

Comment:
Is this a special condition? why are we doing this? would be best to add a comment or put it in some helper / shared package